config/templates/libs/BTPSA-USECASE.json

```diff
@@ -236,11 +240,6 @@
                         "description": "name of the service instance",
                         "title": "name of the service instance"
                     },
-                    "labels":{
```

Comment:
The functionality of  labels are removed with your commit - there seems to be something wrong with the rebase/merge